std/traits.d

```diff
@@ -4072,6 +4072,8 @@ Params:
 Returns:
     Static tuple composed of the members of the enumerated type `E`.
     The members are arranged in the same order as declared in `E`.
+    The name of the enum can be found by querying the compiler for the
```

Comment:
You already have the name, otherwise `case` wouldn't work.

Reply:
As far as I know, we don't: https://run.dlang.io/is/FGGmOH
`EnumMembers` returns an AliasSeq of values. E.g. `members[i].stringof` returns `cast(FooEnum)0`. I might be missing something as I'm no thorough expert.

On the other hand, if we can make it such that we can return the name more easily, then that has my preference as well. But I think that will be a breaking change, as members with the same value are no longer unique.

Reply:
Apparently std.conv: to!string also works to retrieve the member as a string in this example. (It makes me wonder why there is a unittest for the identifier, but that is out of scope for improving the documentation.)

Reply:
```d
import std.conv;
enum Enum {foo = 0, bar = 1, baz = 0}
static foreach(i, member; EnumMembers!Enum)
{
	pragma(msg, to!string(member) ~ " " ~ __traits(identifier, EnumMembers!Enum[i]));
}
```
prints
```
foo foo
bar bar
foo baz
```